suites/gcp.xml

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="GCP">
+    <test name="SmokeTest">
+        <classes>
+            <class name="io.managed.services.test.SmokeTest"/>
+        </classes>
+    </test>
+    <test name="SSOAuthTest">
+        <classes>
+            <class name="io.managed.services.test.SSOAuthTest"/>
+        </classes>
+    </test>     
+    <test name="KafkaCLITest">
+        <classes>
+            <class name="io.managed.services.test.devexp.KafkaCLITest"/>
+        </classes>
+    </test>    
+    <test name="KafkaOperatorTest">
+        <classes>
+            <class name="io.managed.services.test.devexp.KafkaOperatorTest"/>
+        </classes>
+    </test> 
+    <test name="ServiceRegistryOperatorTest">
+        <classes>
+            <class name="io.managed.services.test.devexp.ServiceRegistryOperatorTest"/>
+        </classes>
+    </test> 
+    <test name="KafkaAccessMgmtTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaAccessMgmtTest"/>
+        </classes>
+    </test>
+    <test name="KafkaInstanceAPITest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaInstanceAPITest"/>
+        </classes>
+    </test>
+    <test name="KafkaMgmtAPITest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaMgmtAPITest"/>
+        </classes>
+    </test>
+    <test name="KafkaOverseeingTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaOverseeingTest"/>
+        </classes>
+    </test>
+    <test name="KafkaRegressionTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.KafkaRegressionTest"/>
+        </classes>
+    </test>
+    <test name="LongLiveKafkaInstanceTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.LongLiveKafkaInstanceTest"/>
+        </classes>
+    </test>
+    <!-- QuotaKafkaInstanceTest will be enabled when devExp team resolve issue with token invalidation -->
+    <!-- <test name="QuotaKafkaInstanceTest">
+        <classes>
+            <class name="io.managed.services.test.kafka.QuotaKafkaInstanceTest"/>
+        </classes>
+    </test> -->
+    <test name="CucumberQuickstartsTest">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.CucumberQuickstartsTest"/>
+        </classes>
+    </test>
+    <test name="QuarkusApplicationTest">
+        <classes>
+            <class name="io.managed.services.test.quickstarts.QuarkusApplicationTest"/>
+        </classes>
+    </test>
```

Comment:
will intrfere with tokens untill we resolve it in CLI (so it should be excluded up until then)
```suggestion
    <test name="QuarkusApplicationTest">
        <classes>
            <class name="io.managed.services.test.quickstarts.QuarkusApplicationTest"/>
        </classes>
    </test>
```

Reply:
not resolved ...

Reply:
Removed